data/campaigns/Heir_To_The_Throne/scenarios/17_Scepter_of_Fire.cfg

```diff
@@ -88,6 +90,9 @@
                     [/gold_carryover]
 
                     [note]
+                        [show_if]
+                            {VARIABLE_CONDITIONAL moved_too_close boolean_equals false}
+                        [/show_if]
```

Comment:
Given the implementation of `[objectives]` (in data/lua/wml/objectives.lua), that's kinda surprising... maybe I missed something important though. It's rather complicated.